config/locales/en/providers.yml

```diff
@@ -239,3 +240,17 @@ en:
     success_prospects:
       show:
         h1-heading: What are the prospects of success?
+    limitations:
+      show:
+        h1-heading: What you're covered for
+        h2-heading: Proceeding
+        body: As to proceedings under Part IV Family Law Act 1996 limited to all steps up to and including obtaining and serving a final order and in the event of breach leading to the exercise of a power of arrest to representation on the consideration 	of the breach by the court (but excluding applying for a warrant of arrest, if not attached, and representation in contempt proceedings).
+        link_title: Change
+        form_of_service:
+          html: "<b>Form of service: </b>"
```

Comment:
I don't have a strong opinion about this but it feels a bit weird to have some HTML in here.